<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookWindow</class>
 <widget class="QMainWindow" name="BookWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>535</width>
    <height>601</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>535</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Формат текста</string>
    </property>
    <addaction name="Choice_font_action"/>
    <addaction name="Choice_size_action"/>
    <addaction name="Choice_text_color_action"/>
    <addaction name="Choice_background_color_action"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="Choice_font_action">
   <property name="text">
    <string>Выбрать шрифт</string>
   </property>
  </action>
  <action name="Choice_size_action">
   <property name="text">
    <string>Сменить размер</string>
   </property>
  </action>
  <action name="Choice_text_color_action">
   <property name="text">
    <string>Цвет текста</string>
   </property>
  </action>
  <action name="Choice_background_color_action">
   <property name="text">
    <string>Цвет фона</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
